release/prepareRelease.sh.cmake

```diff
@@ -238,39 +259,56 @@ case "$SYSTEM_NAME" in
 		lua=""
 		case "$SYSTEM_ARCH" in
 			i686|*86)
-				if [ -d /usr/i686-pc-mingw32 ]; then
+				if [ -d /usr/i686-w64-mingw32 ]; then
+					mingw=/usr/i686-w64-mingw32
+				elif [ -d /usr/i686-pc-mingw32 ]; then
 					mingw=/usr/i686-pc-mingw32
 				else
 					mingw=/usr/i686-mingw32
 				fi
 				lua=win32
 			;;
 			x86_64|*64)
-				if [ -d /usr/i686-pc-mingw32 ]; then
+				if [ -d /usr/i686-w64-mingw32 ]; then
+					mingw=/usr/x86_64-w64-mingw32
+				elif [ -d /usr/i686-pc-mingw32 ]; then
 					mingw=/usr/x86_64-pc-mingw32
 				else
 					mingw=/usr/x86_64-mingw32
 				fi
 				lua=win64
 			;;
 		esac
-		
+
 		cp -v ${RTTR_SRCDIR}/contrib/lua/${lua}/lua52.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libgcc_s_sjlj-1.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libminiupnpc-5.dll ${DESTDIR} || exit 1
+
+		if [ -f ${mingw}/bin/libgcc_s_sjlj-1.dll ] ; then
+			cp -v ${mingw}/bin/libgcc_s_sjlj-1.dll ${DESTDIR} || exit 1
+			cp -v ${mingw}/bin/libintl-8.dll ${DESTDIR} || exit 1
+
+			cp -v ${mingw}/bin/libgcc_s_sjlj-1.dll ${DESTDIR}RTTR || exit 1
+			cp -v ${mingw}/bin/zlib1.dll ${DESTDIR}RTTR || exit 1
+			cp -v ${mingw}/bin/libminiupnpc-5.dll ${DESTDIR} || exit 1
+		else
+			cp -v ${mingw}/bin/libgcc_s_seh-1.dll ${DESTDIR} || exit 1
+			cp -v ${mingw}/bin/libstdc++-6.dll ${DESTDIR} || exit
```

Comment:
Hu? libstdc++ instead of libintl and zlib?

Reply:
yep